src/eko/interpolation.py

```diff
@@ -486,13 +396,20 @@ def __init__(self, xgrid, polynomial_degree, log=True, mode_N=True, numba_it=Tru
         # Create blocks
         list_of_blocks = []
         po2 = polynomial_degree // 2
+        # if degree is even, we can not split the block symmetric, e.g. deg=2 -> |-|-|
+        # so, in case of doubt use the block, which lays higher, i.e.
+        # we're not allowed to go so deep -> make po2 smaller
+        if polynomial_degree % 2 == 0:
```

Comment:
why? this is not a bug or problem? and I think having a even `polynomial_degree` is also perfectly fine;

The true bug (previously) was, that we weren't doing what we claimed and what I want to do: "in case of doubt, use the higher block, i.e. the one closer to 1" (see https://n3pdf.github.io/eko/Theory/Interpolation.html)

Reply:
ah, ok, that's fine. I think I misunderstood the comment above